app/assets/admin/css/sass/_components.scss

```diff
@@ -72,8 +72,7 @@
 			vertical-align: top;
 
 			fieldset label {
-				display: block;
-				margin: 0 !important;
+				margin: 0 6px 0 0 !important;
```

Comment:
Please try to avoid the use of `! important`.

Reply:
I haven't add `!important`. I had just removed `display: block.` and give margin of right side.

Reply:
But you have made changes in this line so as I see ` ! important ` is still there please try to remove if possible.